osbrain/agent.py

```diff
@@ -1824,7 +1824,7 @@ def kill(self):
         if self._daemon:
             self._daemon.shutdown()
 
-    def sigint_handler(self, signal, frame):
+    def _sigint_handler(self, signal, frame):
         """
         Handle interruption signals.
         """
```

Comment:
Other candidate methods to set as private:

- `Agent.register`
- `Agent.registered`

Reply:
Missed those 👍